src/main/java/Bundle.properties

```diff
@@ -1648,8 +1648,11 @@ file.share.fileShare.tip=Share this file on your favorite social media networks.
 file.share.fileShare.shareText=View this file.
 file.title.label=Title
 file.citation.label=Citation
-file.citation.notice=This file is part of "{0}". If you use this file, please cite the dataset:
+file.citation.notice=This file is part of "{0}".
```

Comment:
What originally was one message, on one line, is now two messages, on two lines. It unnecessarily takes up more real estate. I'm going to try to clean this up.